physics/smoke_dust/rrfs_smoke_wrapper.meta

```diff
@@ -810,6 +869,30 @@
   type = real
   kind = kind_phys
   intent = inout
+[peak_hr_out]
+ standard_name = peak_hr_fire
+ long_name = hour of peak fire emissions
+ units = none
+ dimensions = (horizontal_loop_extent)
+ type = real
+ kind = kind_phys
+ intent = out
+[lu_nofire_out]
+ standard_name = lu_nofire_out
+ long_name = land use of no fire pixels for type
+ units = frac
```

Comment:
I'm guessing the units should be 1 for this one and the one below.

Reply:
@grantfirl The units will be updated as 1. Thanks.

Reply:
@grantfirl The standard name of lu_nofire is updated to "sum of land use fractions for no fire pixels", and it's unit is fract. Thanks.

Reply:
@grantfirl Sure, I will update the unit to 1.